lang/main.json

```diff
@@ -49,7 +45,7 @@
         "today": "Today"
     },
     "chat": {
-        "error": "Error: your message \"__originalText__\" was not sent. Reason: __error__",
+        "error": "Error: your message \"{{originalText}}\" was not sent. Reason: {{error}}",
```

Comment:
side comment: I think putting the entire message there is not nice. I'd just say "Your message could not be sent".